### PR TITLE
collada-dom 2.5.0

### PR DESCRIPTION
**Pull request by jcarpent**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/collada-dom.rb [hunk 1]
```diff
@@ -30,19 +25,22 @@ def install
...
+      using namespace std;
+
+      int main()
+      {
+        cout << GetCOLLADA_VERSION() << endl;
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-I#{include}/collada-dom2.5", "-L#{lib}", "-lcollada-dom2.5-dp", "-o", "test"
+    assert_equal `./test`.chomp, "1.5.0"
```
Comment (javian) [inline Formula/collada-dom.rb]: thanks for the PR! How come this tests true when the version is 2.5.0 and not 1.5.0 ? Assuming this number will change it wouldn't it be better to do something here that wouldn't change over time so it doesn't have to be updated.

Comment (jcarpent) [reply Formula/collada-dom.rb]: It does not return the current version, but the DAE file version taken in charge. 
So this test will continue with a future release.

Comment (jcarpent) [reply Formula/collada-dom.rb]: @javian Is this answer sufficient? or do you want me to provide another test?

Comment (fxcoudert) [reply Formula/collada-dom.rb]: Thanks for the explanation @jcarpent, it is a good test.  I have added a comment, and I think we're good to go now.

@javian anything else?

Comment (jcarpent) [reply Formula/collada-dom.rb]: Sounds perfect. Thanks for your quick feedbacks!

**Pull request closed without merging**